usnistgov/pfhub
# Nana Ofori-Opoku

Add Nana Ofori-Opoku to community page


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/usnistgov/chimad-phase-field/677)


### How to Review
These changes can be viewed [live](http://random-cat-677.surge.sh/community/).
- Check that you like the look and feel of the Community page.
- Confirm that your name is spelled correctly.
- Confirm that your e-mail address and networking buttons are correct.
- Read your biographical sketch.

#### Remember to tear down the [live site](http://random-cat-677.surge.sh/) when merging or closing this pull-request.
